### ISSUE TITLE
Do we need to add support for EUA functioanlity

### ISSUE THREAD

**Issue by rcrichton** [jembi/openhim-core-js#377]

**Comment by devcritter** [comment]
Do you mean http://wiki.ihe.net/index.php?title=Enterprise_User_Authentication?

**Comment by rcrichton** [comment]
Yup, that's exactly what I mean. I spoke to Justin about this briefly and it provides a mechanisms for users (of point of care systems) to provide a specific assertion about who they are. But I'm not sure we want to perform person level validation, but maybe we should talk about it.
